packages/@aws-cdk/aws-appmesh/test/test.virtual-gateway.ts

```diff
@@ -1,8 +1,10 @@
 import { expect, haveResourceLike } from '@aws-cdk/assert';
+import { Certificate } from '@aws-cdk/aws-certificatemanager';
```

Comment:
As a matter of a general rule, we don't import particular members from outside our module.

```suggestion
import * as acm from '@aws-cdk/aws-certificatemanager';
```